src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -103,12 +117,21 @@ public void periodic() {
 
 
         if (!resetElevator) {
-          rightElevator.getPIDController().setReference(
-            setpoint.get() > kElevator.MAX_POS || setpoint.get() < kElevator.MIN_POS ? kElevator.DEFUALT_VAL : setpoint.get(),
-            CANSparkMax.ControlType.kPosition,
-            0,
-            ff.calculate(0.0)
-          );
+            if(up == true){
+                rightElevator.getPIDController().setReference(
```

Comment:
convert to ternary syntax, so just have one setrefernce function and use smth like

up ? ffu.calctualte : ffd.calcualto

tyrnary syntax is bassicly a simplified if ststamnet ex: (if true) ? (then do this) : (otherwise do this)